sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/ShowTablesSuite.scala

```diff
@@ -119,6 +101,32 @@ trait ShowTablesSuiteBase extends command.ShowTablesSuiteBase {
       assert(errMsg.contains("Database from v1 session catalog is not specified"))
     }
   }
+
+  test("SPARK-34157 Unify output of SHOW TABLES and pass output attributes properly") {
```

Comment:
Let's keep the format consistent next time: `SPARK-34157 :`

Reply:
The consistent format is `SPARK-34157:` w/o any gaps ;-)